.github/workflows/publish.yaml

```diff
@@ -0,0 +1,20 @@
+name: Build and publish pycls to PyPI
+on: push
+jobs:
```

Comment:
does it trigger on other branches?
maybe need
```
on:
  # Trigger the workflow on push or pull request,
  # but only for the master branch
  push:
    branches:
      - master
```

Reply:
Good point, I'm not sure actually. Would be good to specify the branch as you suggest regardless.